tests/base/qprojector.output

```diff
@@ -431,26 +431,26 @@ DEAL:face::0.0 1.0 1.0
 DEAL:face::1.0 1.0 1.0
 DEAL:all::Checking dim 2 1d-points 2
 DEAL:all::Face 0 orientation false
-DEAL:all::0.0 0.0
 DEAL:all::0.0 1.0
+DEAL:all::0.0 0.0
```

Comment:
This one is also interesting: though it didn't cause any problems in practice (except for the Hdiv/Hcurl situation outlined above) this test has printed out the 'wrong' quadrature formula since 2005.

Reply:
Did the code just forget to flip the direction of the edge?